services/ui-src/src/components/report/ReportPageWrapper.tsx

```diff
@@ -103,15 +105,35 @@ export const ReportPageWrapper = () => {
                     Previous
                   </Button>
                 )}
-                {parentPage.index < parentPage.childPageIds.length - 1 && (
+                {parentPage.index < parentPage.childPageIds.length - 1 &&
+                  parentPage.index !== 0 && (
+                    <Button
+                      onClick={() => SetPageIndex(parentPage.index + 1)}
+                      marginLeft="auto"
+                    >
+                      Continue
+                    </Button>
+                  )}
+                {parentPage.index == 0 && (
                   <Button
                     onClick={() => SetPageIndex(parentPage.index + 1)}
                     marginLeft="auto"
+                    justifyContent={"flex-end"}
                   >
                     Continue
                   </Button>
                 )}
               </Flex>
+              <Box flex="auto" width="97.5%">
```

Comment:
What's the consensus here on half percentages? This was the perfect number to get the text to match and the words to align as shown in [Figma](https://www.figma.com/design/B8IEoYKQ7DuQyx5nStls8F/HCBS?node-id=1-2&node-type=canvas&t=ikaqfqzeKN7Uki1Y-0).

Reply:
Should this be a set margin instead? Might be worth checking

Reply:
Removed. Per convo with Aileen